app-laptop/tpacpi-bat/files/tpacpi-bat.confd.0

```diff
@@ -3,6 +3,9 @@
 # use '/etc/init.d/tpacpi-bat info' at any time to check the thresholds
 # currently used by the driver
 
+# number of batteries you have
```

Comment:
Documentation is wrong, `BATS` is supposed to be a list of batteries, e.g. `BATS="1 3"`.